docs/troubleshooting.md

```diff
@@ -0,0 +1,130 @@
+===================
+
+1. [No packages show up in the UI](#no-packages-show-up-in-the-ui-(no-operatorhub-items-found))
+2. [OperatorSource failing to download from datastore](#operatorSource-failing-to-download-from-datastore)
+3. [OperatorSource `MESSAGE` reads unknown error (status 404)](#operatorsource-`message`-reads-unknown-error-(status-404))
+4. [Cannot uninstall operator from the CLI in a clean way](#cannot-uninstall-operator-from-the-cli-in-a-clean-way) 
+5. [CatalogSourceConfig stuck in `Configuring` phase](#catalogsourceconfig-stuck-in-configuring-phase)
+
+
+## No packages show up in the UI (No OperatorHub Items Found)
+
+When you install an Openshift cluster, the embedded Marketplace (OperatorHub) comes with a default list of operators that can be installed onto the cluster. A number of resources need to be investigated if however you see a message like this: 
+
+```
+No OperatorHub Items found 
+
+Please check that the OperatorHub is running and that you have created a valid OperatorSource. For more information about OperatorHub, please click here. 
+```
+
+First, investigate the OperatorSources. 
+
+```
+$ oc get opsrc -n openshift-marketplace
+
+NAME                  TYPE          ENDPOINT              REGISTRY              DISPLAYNAME           PUBLISHER   STATUS      MESSAGE                                       AGE
+certified-operators   appregistry   https://quay.io/cnr   certified-operators   Certified Operators   Red Hat     Succeeded   The object has been successfully reconciled   22h
+community-operators   appregistry   https://quay.io/cnr   community-operators   Community Operators   Red Hat     Succeeded   The object has been successfully reconciled   22h
+redhat-operators      appregistry   https://quay.io/cnr   redhat-operators      Red Hat Operators     Red Hat     Succeeded   The object has been successfully reconciled   22h
+```
+
+If the status of any of the OperatorSource is anything other than `Succeeded`, you can investigate the OperatorSource with 
+`oc describe opsrc <Name-of-OperatorSource> -n openshift-marketplace`. [This section]() of the doc also contains helpful information about debugging an OperatorSource. 
+
+If the OperatorSources look healthy, check that the CatalogSourceConfigs for the corresponding OperatorSources were created.
+
+```
+$ oc get csc -n openshift-marketplace
+
+NAME                  STATUS      MESSAGE                                       AGE
+certified-operators   Succeeded   The object has been successfully reconciled   23h
+community-operators   Succeeded   The object has been successfully reconciled   23h
+redhat-operators      Succeeded   The object has been successfully reconciled   23h
+```
+
+If the status of any of the CatalogSourceConfigs is anything other than `Succeeded`, you can inspect the CatalogSourceConfig with `oc describe csc <Name-of-CatalogSourceConfig>`. [This section]() of the doc also contains helpful information about debugging a CatalogSourceConfig. 
```

Comment:
`This section` has no link

Reply:
@galletti94 thank you for noticing this. Addressed.